firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandlerAutomated.cs

```diff
@@ -200,7 +206,20 @@ public class UIHandlerAutomated : UIHandler {
         testRunner.Update();
       }
     }
+    
+    // Set `USE_FIRESTORE_EMULATOR` environment variable and port number to 
+    // run tests against Firestore emulator.
+    private void SetFirestoreEmulatorEnv(string port="8080") {
```

Comment:
When are these methods used? Normally environment variables are set from outside program (i.e. within a script that runs test, or IDE config.)